README.md

```diff
@@ -22,6 +22,8 @@
 15. Работоспособность при выключенном (незагруженном) JavaScript
 16. Работоспособность при выключенном Flash
 17. Отсутствие багов при увеличенном шрифте
+18. Write clear code don't mess the code.
```

Comment:
It's checklist, not guidelines. So every rule is something we can simple check. "Write clean code" isn't a rule which could be verified